modules/react-native-status/android/build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
     implementation 'com.instabug.library:instabug:3+'
     implementation 'status-im:function:0.0.1'
 
-    String statusGoVersion = 'develop-g19b53030'
+    String statusGoVersion = 'develop-g5ef26596'
```

Comment:
maybe we should use `0.13.0` release of status-go?
see: https://github.com/status-im/status-react/issues/5643